files/default/sql/ddl/3.4.0__initial_tables.sql

```diff
@@ -2257,3 +2257,29 @@ CREATE TABLE `serving_key` (
                                CONSTRAINT `feature_view_serving_key_fk` FOREIGN KEY (`feature_view_id`) REFERENCES `feature_view` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION,
                                CONSTRAINT `feature_group_serving_key_fk` FOREIGN KEY (`feature_group_id`) REFERENCES `feature_group` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION
 ) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
+
+CREATE TABLE `model` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `name` VARCHAR(255) NOT NULL,
+  `project_id` int(11) NOT NULL,
+  UNIQUE KEY `project_unique_name` (`name`, `project_id`),
+  PRIMARY KEY (`id`),
+  CONSTRAINT `model_project_fk` FOREIGN KEY (`project_id`) REFERENCES `project` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION
+) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
+
+CREATE TABLE `model_version` (
+  `model_id` int(11) NOT NULL,
+  `version` int(11) NOT NULL,
+  `user_full_name` VARCHAR(255) DEFAULT NULL,
+  `created` timestamp DEFAULT NULL,
+  `description` VARCHAR(255) DEFAULT NULL,
+  `metrics` VARCHAR(1000) DEFAULT NULL,
+  `program` VARCHAR(1000) DEFAULT NULL,
+  `framework` VARCHAR(1000) DEFAULT NULL,
+  `environment` VARCHAR(1000) DEFAULT NULL,
+  `experiment_id` VARCHAR(1000) DEFAULT NULL,
+  `experiment_project_name` VARCHAR(100) DEFAULT NULL,
+  `model_registry_id` int(11) NOT NULL,
```

Comment:
shouldn't the model registry id be part of the model?
also maybe the user full name should be a user id foreign key?

Reply:
I'm not sure I will keep the model_registry_id, because it's pretty much just the project_id so would not make sense to store it I think.

Reply:
On the user full name I just dumped what we had in elastic there for now. I was thinking it would be better to store the user FK there instead as well but started thinking of how to migrate it first. Will look more at it later on when closer to merging.

Reply:
I changed the user_full_name to be a user_id foreign key and added the migration in expat. Also the model_registry_id is removed and is now simply the project_id and not stored in the database.